From ded050bca1992e57147e86986acbf2a6c46fd91c Mon Sep 17 00:00:00 2001
From: Maximilian Luz <luzmaximilian@gmail.com>
Date: Tue, 30 Dec 2025 23:42:38 +0100
Subject: [PATCH 6/6] Drop riscv64 configs

Those seem to make some problems... so drop them for now.
---
 redhat/configs/priority.common | 4 ----
 redhat/configs/priority.fedora | 6 ------
 redhat/configs/priority.rhel   | 4 ----
 redhat/kernel.spec.template    | 7 -------
 4 files changed, 21 deletions(-)

diff --git a/redhat/configs/priority.common b/redhat/configs/priority.common
index ea4b1122b1a1..cb1f09840cff 100644
--- a/redhat/configs/priority.common
+++ b/redhat/configs/priority.common
@@ -19,7 +19,3 @@ aarch64=generic:generic-arm:generic-arm-aarch64
 aarch64-debug=generic:generic-arm:generic-arm-aarch64:debug:debug-arm-aarch64
 aarch64-16k-debug=generic:generic-arm:generic-arm-aarch64:generic-arm-aarch64-16k:debug:debug-arm-aarch64:debug-arm-aarch64-16k
 aarch64-64k-debug=generic:generic-arm:generic-arm-aarch64:generic-arm-aarch64-64k:debug:debug-arm-aarch64:debug-arm-aarch64-64k
-
-# riscv64
-riscv64=generic:generic-riscv:generic-riscv-riscv64
-riscv64-debug=generic:generic-riscv:generic-riscv-riscv64:debug:debug-riscv-riscv64
diff --git a/redhat/configs/priority.fedora b/redhat/configs/priority.fedora
index e72726642039..658970a57293 100644
--- a/redhat/configs/priority.fedora
+++ b/redhat/configs/priority.fedora
@@ -42,9 +42,3 @@ aarch64-rt=generic:generic-arm:generic-arm-aarch64:rt-generic:rt-generic-arm:rt-
 aarch64-rt-debug=generic:generic-arm:generic-arm-aarch64:debug:debug-arm:debug-arm-aarch64:rt-generic:rt-generic-arm:rt-generic-arm-aarch64:rt-debug:rt-debug-arm:rt-debug-arm-aarch64
 aarch64-rt-64k=generic:generic-arm:generic-arm-aarch64:rt-generic:rt-arm-aarch64:rt-arm-aarch64-64k
 aarch64-rt-64k-debug=generic:generic-arm:generic-arm-aarch64:rt-generic:rt-arm-aarch64:rt-arm-aarch64-64k:debug:debug-arm-aarch64:rt-debug
-
-# riscv64
-riscv64=generic:generic-riscv:generic-riscv-riscv64
-riscv64-debug=generic:generic-riscv:generic-riscv-riscv64:debug:debug-riscv-riscv64
-riscv64-rt=generic:generic-riscv:generic-riscv-riscv64:rt-generic:rt-generic-riscv-riscv64
-riscv64-rt-debug=generic:generic-riscv:generic-riscv-riscv64:debug:debug-riscv-riscv64:rt-generic:rt-generic-riscv-riscv64:rt-debug:rt-debug-riscv-riscv64
diff --git a/redhat/configs/priority.rhel b/redhat/configs/priority.rhel
index 2f739fc47f31..4e56dcad5f76 100644
--- a/redhat/configs/priority.rhel
+++ b/redhat/configs/priority.rhel
@@ -46,7 +46,3 @@ aarch64-rt-64k=generic:generic-arm:generic-arm-aarch64:rt-generic:rt-arm-aarch64
 aarch64-rt-64k-debug=generic:generic-arm:generic-arm-aarch64:rt-generic:rt-arm-aarch64:rt-arm-aarch64-64k:debug:debug-arm-aarch64:rt-debug
 aarch64-automotive=generic:generic-arm:generic-arm-aarch64:rt-generic:rt-generic-arm:rt-generic-arm-aarch64:automotive-generic:automotive-generic-arm:automotive-generic-arm-aarch64
 aarch64-automotive-debug=generic:generic-arm:generic-arm-aarch64:debug:debug-arm:debug-arm-aarch64:rt-generic:rt-generic-arm:rt-generic-arm-aarch64:rt-debug:rt-debug-arm:rt-debug-arm-aarch64:automotive-generic:automotive-generic-arm:automotive-generic-arm-aarch64:automotive-debug:automotive-debug-arm:automotive-debug-arm-aarch64
-
-# riscv64
-riscv64=generic:generic-riscv:generic-riscv-riscv64
-riscv64-debug=generic:generic-riscv:generic-riscv-riscv64:debug:debug-riscv-riscv64
diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 79e67a8b1440..be4e401212b5 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1015,9 +1015,6 @@ Source33: %{name}-x86_64-debug-rhel.config
 # ARM64 64K page-size kernel config
 Source42: %{name}-aarch64-64k-rhel.config
 Source43: %{name}-aarch64-64k-debug-rhel.config
-
-Source44: %{name}-riscv64-rhel.config
-Source45: %{name}-riscv64-debug-rhel.config
 %endif
 
 %if %{include_rhel} || %{include_automotive}
@@ -1039,8 +1036,6 @@ Source58: %{name}-s390x-fedora.config
 Source59: %{name}-s390x-debug-fedora.config
 Source60: %{name}-x86_64-fedora.config
 Source61: %{name}-x86_64-debug-fedora.config
-Source700: %{name}-riscv64-fedora.config
-Source701: %{name}-riscv64-debug-fedora.config
 
 Source62: def_variants.yaml.fedora
 %endif
@@ -1125,8 +1120,6 @@ Source482: %{name}-aarch64-rt-64k-fedora.config
 Source483: %{name}-aarch64-rt-64k-debug-fedora.config
 Source484: %{name}-x86_64-rt-fedora.config
 Source485: %{name}-x86_64-rt-debug-fedora.config
-Source486: %{name}-riscv64-rt-fedora.config
-Source487: %{name}-riscv64-rt-debug-fedora.config
 %endif
 %endif
 
-- 
2.52.0

